diff --git a/BUILD.gn b/BUILD.gn
index 5833911122..a38a194c31 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -38,7 +38,7 @@ if (!build_with_chromium) {
   # 'ninja default' and then 'ninja all', the second build should do no work.
   group("default") {
     testonly = true
-    deps = [ ":webrtc" ]
+    deps = [ ":webrtc","//libwebrtc" ]
     if (rtc_build_examples) {
       deps += [ "examples" ]
     }
